public/js/applicantModal.js

```diff
@@ -50,7 +49,8 @@ function copyToClipboard(element) {
     document.querySelector("button.clipboard").innerText = 'Copied';
 }
 
-$(document).ready(function () {
+export function displayApplicantModal() {
```

Comment:
rename to `addEventListenersToDisplayApplicantModal`

Reply:
done